banker.csv

```diff
@@ -112,4 +112,16 @@ OrgNummer,Navn,Filnavn,EndepunktProduksjon,EndepunktTest
 930050237,Sparebank 1 Finans Nord-Norge AS,Sparebank_1_Finans_Nord-Norge_AS_930050237_Accounts-API.json,https://api.sparebank1.no/dsopaccountcontrolinfo/v2/AccountControlInfoService/v2/930050237,
 975966453,SPAREBANK 1 KREDITT AS,Sparebank_1_KREDITT_AS_975966453_Accounts-API.json,https://api.sparebank1.no/dsopaccountcontrolinfo/v2/AccountControlInfoService/v2/975966453,https://api-test.sparebank1.no/dsopaccountcontrolinfo/v2/AccountControlInfoService/v2/975966453,
 984150865,Resurs bank AB,Resurs_bank_AB_984150865_Accounts-API.json,,https://apigw-integration.test.resurs.loc/api/dsop_account_service
-946670081,Sparebanken Sogn og Fjordane,Sparebanken_Sogn_og_Fjordane_946670081_Accounts-API.json,https://bf-esb-internet.edb.com/secesb/rest/era-dsop/v1/3702,
\ No newline at end of file
+946670081,Sparebanken Sogn og Fjordane,Sparebanken_Sogn_og_Fjordane_946670081_Accounts-API.json,https://bf-esb-internet.edb.com/secesb/rest/era-dsop/v1/3702,
+982719445, Jernbanepersonalets bank og forsikring,{hva skal være her - gjelder alle under}, https://bf-esb-internet.edb.com/secesb/rest/era-dsop/v1/1449
+937897375, Tysnes Sparebank, , https://bf-esb-internet.edb.com/secesb/rest/era-dsop/v1/3528
+937899963, Sunndal Sparebank, ,https://bf-esb-internet.edb.com/secesb/rest/era-dsop/v1/4039
+937900953, Rindal Sparebank, ,https://bf-esb-internet.edb.com/secesb/rest/era-dsop/v1/4113
+937901569, Oppdalsbanken, ,https://bf-esb-internet.edb.com/secesb/rest/era-dsop/v1/4269
+947278770, Orkla Sparebank, ,https://bf-esb-internet.edb.com/secesb/rest/era-dsop/v1/4274
+937902174, Hemne Sparebank(Trøndelag Sparebank), ,https://bf-esb-internet.edb.com/secesb/rest/era-dsop/v1/4314
+937903979, Sparebanken Narvik, ,https://bf-esb-internet.edb.com/secesb/rest/era-dsop/v1/4529
+989997254, Sparesmart (En del av Eika Kredittbank), ,https://bf-esb-internet.edb.com/secesb/rest/era-dsop/v1/9815
```

Comment:
Fjern ekstra whitespace, legg til filnavn og et avsluttende komma for å tydeliggjøre at det mangler testendepunkt